usaspending_api/search/tests/test_spending_by_award_type_count.py

```diff
@@ -1,16 +1,70 @@
 import json
-
 import pytest
-from rest_framework import status
 
-# Third-party app imports
 from django_mock_queries.query import MockModel
+from model_mommy import mommy
+from rest_framework import status
 
-# Imports from your apps
 from usaspending_api.common.helpers.unit_test_helper import add_to_mock_objects
 from usaspending_api.search.tests.test_mock_data_search import all_filters
 
 
+@pytest.fixture
+def award_data_fixture(db):
+    mommy.make("awards.TransactionNormalized", id=2)
+    mommy.make("references.LegalEntity", legal_entity_id=20)
+    mommy.make(
+        "awards.Award",
+        base_and_all_options_value=None,
+        base_exercised_options_val=None,
+        category="loans",
+        certified_date="2018-07-12",
+        create_date="2018-02-03 18:54:06.983782+00",
+        data_source="DBR",
+        date_signed="2009-09-10",
+        description="FORD MOTOR COMPANY",
+        fain="DECF0000058",
+        fiscal_year=2009,
+        fpds_agency_id=None,
+        fpds_parent_agency_id=None,
+        funding_agency_id=None,
+        generated_unique_award_id="ASST_NON_DECF0000058_8900",
+        id=48518634,
+        is_fpds=False,
+        last_modified_date="2018-08-02",
+        latest_transaction_id=2,
+        non_federal_funding_amount=0,
+        parent_award_piid=None,
+        period_of_performance_current_end_date="2039-09-09",
+        period_of_performance_start_date="2009-09-10",
+        piid=None,
+        potential_total_value_of_award=None,
+        recipient_id=20,
+        subaward_count=0,
+        total_funding_amount=5907041570,
+        total_loan_value=5907041570,
+        total_obligation=5907041570,
+        total_outlay=None,
+        total_subaward_amount=None,
+        total_subsidy_cost=3000186413,
+        transaction_unique_id="A000_8900_DECF0000058_-NONE-",
+        type="07",
+        type_description="DIRECT LOAN (E)",
+        uri=None,
+    )
+
+# ===================================================
+# Below tests SKIPPED due to the introduction of asyncpg.
+# asyncpg is a different library to open DB connections to Postgres and supports async functions
+# The test data fixtures hold an open idle DB transaction which blocks SQL queries using the new connection
+# These test need to be re-implemented, but the team has allowed this feature to move forward
+# As the task is much larger than anticipated
+
+# Tony Aug 2019
+# ===================================================
```

Comment:
# Tony 2019
seriously, good documentation though. That will come in handy in a year.